src/library_wasm_worker.js

```diff
@@ -212,7 +212,7 @@ mergeInto(LibraryManager.library, {
   // see https://bugs.chromium.org/p/chromium/issues/detail?id=1167541
   // https://github.com/tc39/proposal-atomics-wait-async/blob/master/PROPOSAL.md
   // This polyfill performs polling with setTimeout() to observe a change in the target memory location.
-  emscripten_atomic_wait_async__postset: "if (!Atomics['waitAsync'] || parseInt((navigator.userAgent.match(/Chrom(e|ium)\\/([0-9]+)\\./)||[])[2]) < 91) { \n"+
+  emscripten_atomic_wait_async__postset: "if (!Atomics['waitAsync'] || jstoi_q((navigator.userAgent.match(/Chrom(e|ium)\\/([0-9]+)\\./)||[])[2]) < 91) { \n"+
```

Comment:
Why replace `parseInt` with a function that just calls `parseInt`? I'm not sure why `jstoi_q` exists... does it avoid a closure warning somehow? (I don't see an explanation on the definition of `jstoi_q`) If that's not it, how does it relate to the main change here?

Reply:
Yes, its simply a wrapper around `parseInt` that only takes one argument (and therefore uses radix guessing).       Calling parsing directly with just one argument is a closure warning: see #10457 for this is needed.

This fix a closure warning.. which is part of what this change does.